knative-eventing/eventing.addon

```diff
@@ -3,8 +3,8 @@
 # OpenShift-Version: >=3.11.0
 # Minishift-Version: >=1.25.0
 # Depends-On: anyuid
-# Required-Vars: KNATIVE_EVENTING_VERSION
-# Var-Defaults: KNATIVE_EVENTING_VERSION=v0.2.0
+# Required-Vars: KNATIVE_VERSION
+# Var-Defaults: KNATIVE_VERSION=v0.2.0
```

Comment:
Actually, as long as we have this as the default, we can just remove the suggestion of setting it in the profile in the README. Maybe... I can't decide. :)